### ISSUE TITLE
"libmkl_rt" on Windows

### ISSUE THREAD

**Issue by jonas-kr** [JuliaMath/FFTW.jl#120]
Using FFTW with JULIA_FFTW_PROVIDER = MKL on Windows 10 fails on my system.

```
ERROR: LoadError: LoadError: could not load library "C:\Users\jonas\.julia\conda\3\Library\bin\libmkl_rt"
The specified module could not be found.

Stacktrace:
 [1] top-level scope at C:\Users\jonas\.julia\packages\FFTW\MJ7kl\src\fft.jl:58
 [2] include at .\boot.jl:328 [inlined]
 [3] include_relative(::Module, ::String) at .\loading.jl:1094
 [4] include at .\Base.jl:31 [inlined]
 [5] include(::String) at C:\Users\jonas\.julia\packages\FFTW\MJ7kl\src\FFTW.jl:1
 [6] top-level scope at C:\Users\jonas\.julia\packages\FFTW\MJ7kl\src\FFTW.jl:77
 [7] include at .\boot.jl:328 [inlined]
 [8] include_relative(::Module, ::String) at .\loading.jl:1094
 [9] include(::Module, ::String) at .\Base.jl:31
 [10] top-level scope at none:2
 [11] eval at .\boot.jl:330 [inlined]
 [12] eval(::Expr) at .\client.jl:432
 [13] top-level scope at .\none:3
in expression starting at C:\Users\jonas\.julia\packages\FFTW\MJ7kl\src\fft.jl:58
in expression starting at C:\Users\jonas\.julia\packages\FFTW\MJ7kl\src\FFTW.jl:77
ERROR: Failed to precompile FFTW [7a1cc6ca-52ef-59f5-83cd-3a7055c09341] to C:\Users\jonas\.julia\compiled\v1.2\FFTW\PvIn2.ji.
Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at .\loading.jl:1253
 [3] _require(::Base.PkgId) at .\loading.jl:1013
 [4] require(::Base.PkgId) at .\loading.jl:911
 [5] require(::Module, ::Symbol) at .\loading.jl:906
```

However, I've been able to fix the issue by replacing all occurrences of "libmkl_rt" in ".../deps/build.jl" and ".../src/FFTW.jl" with the new variable mkllib

```
if Sys.iswindows()
    mkllib = "mkl_rt"
else
    mkllib = "libmkl_rt"
end
```

According to the article [A New Linking Model-Single Dynamic Library mkl_rt Since Intel® MKL 10.3](https://software.intel.com/en-us/articles/a-new-linking-model-single-dynamic-library-mkl_rt-since-intel-mkl-103), the mkl library has a different name in Windows operating systems.

**Comment by senpai-a** [comment]
I encounterd the same problem, thank u for your fix.
also I got
`Intel MKL FATAL ERROR: Cannot load mkl_intel_thread.dll`
and then Julia crashes. I need to add "C:\users\username\.julia\conda\3\Library\bin" to Path to fix this.

**Comment by ararslan** [comment]
Fixed by #128.
